lib/perl/Genome/Config/AnalysisProject/Command/CopyConfig.t

```diff
@@ -51,6 +54,7 @@ my $target_project = Genome::Config::AnalysisProject->create(
 my $cmd = $class->create(
     to_project => $target_project,
     from_project => $source_project,
+    tags => 1,
```

Comment:
I reworked the test so the flags are parametrized.  That way it wasn't a bunch of extra coding to try a bunch of combinations.  It still runs quickly, so I'm inclined to err on the side of comprehensiveness.